server/commands/hubs.py

```diff
@@ -32,6 +32,7 @@
     "ooc_cmd_arup_disable",
     "ooc_cmd_toggle_getareas",
     "ooc_cmd_toggle_spectate",
+    "ooc_cmd_toggle_passingmsg",
```

Comment:
Not too big on the command name, maybe /toggle_passing_ic?

Reply:
Or at least /toggle_passing_msg

Reply:
Tbh I'm not sure which one sounds better. I left it as /toggle_passing_ic, anyway